pkgs/development/compilers/kind2/default.nix

```diff
@@ -1,33 +1,59 @@
 { lib
 , rustPlatform
 , fetchCrate
+, fetchpatch
 , stdenv
 , darwin
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "kind2";
-  version = "0.3.7";
+  version = "0.3.10";
 
   src = fetchCrate {
     inherit pname version;
-    sha256 = "sha256-ZG0BbGcjQBqeNTqfy7WweVHK7sUuKeQSsFi9KIsyIE4=";
+    hash = "sha256-X2sjfYrSSym289jDJV3hNmcwyQCMnrabmGCUKD5wfdY=";
   };
 
-  cargoSha256 = "sha256-j64L3HNk2r+MH9eDHWT/ARJ9DT4CchcuVxtIYYVsDxo=";
+  cargoHash = "sha256-nWL2ss04tr93ViNmGBiLoBZZ4az1wuChbsKipcjK+/c=";
+
+  depsExtraArgs.postBuild =
```

Comment:
Why is this not using cargoPatches or postPatch?

Reply:
this is patching a vendored dependency, so `cargoPatches` wouldn't work

I think I messed up something when I was trying to use `postPatch` initially, I've switched back to `postPatch` and it works now